### PR TITLE
Added bootstrap dependency and created nav bar.

### LINKED ISSUE #31: implement twitter bootstrap

### PR DESCRIPTION
**Pull request by mbockus**
Added the bootstrap dependency. Also created a collapsible nav bar that should be responsive to mobile. This PR should resolve #31.

### CODE REVIEW

**Comment by Mike-Loffland** [comment]
:thumbsup:

File: school-finder-frontend/app/components/nav/NavController.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+(function () {
+  var sfApp = angular.module('sfApp');
+
+  var NavCtrl = function($scope, $location){
+      $scope.isActive = function (viewLocation) {
+          return viewLocation === $location.path().replace(/^(?:\/)+/, "");;
```
Comment (DevinClark) [inline school-finder-frontend/app/components/nav/NavController.js]: I believe UI-sref-active could replace this function. http://angular-ui.github.io/ui-router/site/#/api/ui.router.state.directive:ui-sref-active 
Also, Two semicolons.

Comment (mbockus) [inline school-finder-frontend/app/components/nav/NavController.js]: @DevinClark Good catch! I was hoping there was a better way to determine the active route. The PR has been updated.

**Comment by DevinClark** [comment]
Awesome! You can remove the href too. UI-Router adds this for you with ui-sref

**Comment by mbockus** [comment]
Ah, very cool. PR is updated.

**Comment by joekarl** [comment]
This looks good, as a side note, this doesn't work directly in chrome due to same origin issues.
Not a big deal, but something to be aware of.

+1

**Pull request merged**